controllers/grid/settings/roles/form/UserGroupForm.php

```diff
@@ -257,13 +257,12 @@ public function execute(...$functionParams)
     public function _assignStagesToUserGroup($userGroupId, $userAssignedStages)
     {
         $contextId = $this->getContextId();
-        $userGroupDao = DAORegistry::getDAO('UserGroupDAO'); /** @var UserGroupDAO $userGroupDao */
 
         // Current existing workflow stages.
         $stages = WorkflowStageDAO::getWorkflowStageTranslationKeys();
 
         foreach (array_keys($stages) as $stageId) {
-            $userGroupDao->removeGroupFromStage($contextId, $userGroupId, $stageId);
+           Repo::userGroup()->removeGroupFromStage($contextId, $userGroupId, $stageId);
```

Comment:
Wrong indentation (I suspect the code formatter isn't running as it should prior to commits)

Reply:
Fixed